app/src/main/java/com/asuc/asucmobile/domain/fragments/CalendarFragment.java

```diff
@@ -0,0 +1,120 @@
+package com.asuc.asucmobile.domain.fragments;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+
+import com.asuc.asucmobile.R;
+import com.asuc.asucmobile.adapters.CalendarAdapter;
+import com.asuc.asucmobile.models.CalendarItem;
+import com.asuc.asucmobile.utilities.NavigationGenerator;
+
+import java.util.ArrayList;
+
+public class CalendarFragment extends Fragment {
+
+    ArrayList<CalendarItem> calendarItems;
+    private static CalendarAdapter adapter;
+    private ListView listView;
+
+    private static final String ARG_SECTION_NUMBER = "calendar_number";
+
+    public CalendarFragment() {
+    }
+
+    /**
+     * Returns a new instance of this fragment for the given section
+     * number.
+     */
+    public static CalendarFragment newInstance(int sectionNumber) {
+        CalendarFragment fragment = new CalendarFragment();
+        Bundle args = new Bundle();
+        args.putInt(ARG_SECTION_NUMBER, sectionNumber);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    private static final String[] info_calendar = new String[] {"L&S Deadlines", "Academic Calendar"};
```

Comment:
best practice: these belong in `values` as Lombok utility classes

Reply:
weren't we gonna pull this from firebase?

Reply:
nvm addressed